src/run_build.py

```diff
@@ -43,6 +42,12 @@ def main():
             manifest.to_file(args.ref_manifest)
         exit(0)
 
+    output_dir = os.path.join(
```

Comment:
Consolidate into `args.output_dir` or another class so we don't duplicate this code.